.github/workflows/fill_station_build_and_push.yml

```diff
@@ -0,0 +1,112 @@
+name: Fill Station Build Test Push
+
+# Configures this workflow to run every time a change is pushed to the branch called `main` or a pull request to `main` is created
+on:
+  push:
+    branches: ['main']
+  pull_request:
+    branches: ['main']
+
+# Defines two custom environment variables for the workflow. These are used for the Container registry domain, and a name for the Docker image that this workflow builds.
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository_owner }}/fill-station
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, which is the CWD for
+      # the rest of the steps
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - uses: bazel-contrib/setup-bazel@0.8.5
+        with:
+          # Avoid downloading Bazel every time.
+          bazelisk-cache: true
+          # Store build cache per workflow.
+          disk-cache: ${{ github.workflow }}
+          # Share repository cache between workflows.
+          repository-cache: true
+
+        # test
+      - name: Build and Test with Bazel
+        run: bazel --output_base=/tmp/bazel/output test //fill:all
```

Comment:
Also doesn't this just test fill:all? Does it also build it?

Reply:
Bazel test also builds it (otherwise there's no guarantee there's anything to test)